operators/pkg/instctrl/cloudinit_test.go

```diff
@@ -65,10 +66,9 @@ var _ = Describe("Generation of the cloud-init configuration", func() {
 		environmentName   = "control-plane"
 		tenantName        = "tester"
 
-		nextcloudBaseURL  = "https://nextcloud.example.com"
-		webdavCredentials = "webdav-credentials"
-		webdavUsername    = "username"
-		webdavPassword    = "password"
+		NFSServiceName = "rook-ceph-nfs-my-nfs-a.rook-ceph.svc.cluster.local"
```

Comment:
Keep test constants simple, they do not need to fully respect reality.

Reply:
Simplified